### DISCUSSION TITLE
How to enable relative dates?

### DISCUSSION THREAD

**Discussion by farzadmf** [eza-community/eza#639] [Q&A, Howto, Help] [answered]
Hey everyone,

I'm having a hard time figuring out how to enable relative dates. Is it supposed to be enabled by default? For example, I do `eza --long`, and dates are "absolute".

Tried searching through the command-line options, but I didn't find anything.

Thank you

**Comment by cafkafk** [comment, marked as answer]
You probably just need `eza -l --time-style relative`,  but I use it with `eza -l --time-style relative --sort time` as well mostly. Hope this helps :3

**Reply by farzadmf** [reply]
That helps for sure. Thank you @cafkafk
